postcss.config.js

```diff
@@ -1,17 +1,18 @@
 module.exports = {
   plugins: [
-    "postcss-import",
-    "postcss-nested",
-    "postcss-flexbugs-fixes",
+    'postcss-import',
```

Comment:
Why the single quotes :see_no_evil:

Reply:
@jfranciscosousa , I think that's my prettier. As Pedro said, here we go 😅...

Reply:
@jfranciscosousa , I think that's my prettier. As Pedro said, here we go 😅...